src/masoniteorm/collection/Collection.py

```diff
@@ -275,12 +275,18 @@ def put(self, key, value):
         self[key] = value
         return self
 
-    def random(self):
+    def random(self, count=None):
```

Comment:
Did not want to define count=1 by default, as I would like the behaviour described in the PR:
- count is not specified  => returns one element
- count is specified => returns a collection (event if 1)